exporter/elasticsearchexporter/README.md

```diff
@@ -213,3 +213,89 @@ Other metric types (Histogram, Exponential Histogram, Summary) are ignored.
 [data stream]: https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html
 [ecs]: https://www.elastic.co/guide/en/ecs/current/index.html
 [SemConv]: https://github.com/open-telemetry/semantic-conventions
+
+
+## ECS Mapping
+
+When `mode` is set to `ecs`, `elasticsearchexporter` performs conversions for resource-level attributes from their Semantic Conventions (SemConv) names to equivalent Elastic Common Schema (ECS) names.
+
+If the target ECS field name is specified as an empty string (""), the converter will neither convert the SemConv key to the equivalent ECS name nor pass through the SemConv key as-is to become the ECS name.
+
+"Preserved" means the convention should be preserved in ECS mode. This can happen when an attribute needs to be mapped to an ECS equivalent but also preserved in its original form.
```

Comment:
Do you mean that for `host.name`, we will duplicate the attribute to `host.hostname`?

Is this treatment only true for the `host.name` attribute? Can this be handled as a separate paragraph?

Reply:
so far it is just for host.hostname.
i went with this format just in case we have more exceptions later

Reply:
the column header says "Preserve" but you use the word "Preserved". If this is exceptional treatment, maybe it needs to be called out more. I also don't understand exactly what you write. Do you mean this?
```suggestion
When "Preserved" is true, the attribute will be preserved in the payload and duplicated as mapped to its ECS equivalent.
```